integration/setup/common.go

```diff
@@ -0,0 +1,60 @@
+package setup
+
+import (
+	"context"
+
+	corev1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/e2e-harness/pkg/release"
+	"github.com/giantswarm/e2etemplates/pkg/chartvalues"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/azure-operator/integration/env"
+)
+
+func Common(config Config) error {
```

Comment:
A comment on what does this function would be nice.

Reply:
Then please provide a suggestion. This is so obvious to me personally that I do not know what to write here as everything I could come up with would be rather useless. Please also suggest for the one below.

Reply:
common installs components required to run the operator.

provider installs the operator and tenant cluster CR.